src/lib.rs

```diff
@@ -700,7 +828,7 @@ mod test {
 
     #[test]
     fn test_zero_legacy_002_hash() {
-        run_hash_test::<AccessListTransaction>("./test/zero_eip_2718_002.json");
```

Comment:
Whoops!

Reply:
I fixed this and other similar misconfigs in tests